Themis/ViewModels/Exercise/ExerciseViewModel.swift

```diff
@@ -85,6 +89,14 @@ class ExerciseViewModel: ObservableObject {
         || ((exam?.isOver ?? false) && !(exam?.isAssessmentDue ?? true))
     }
     
+    var isSecondCorrectionRoundEnabled: Bool {
+        exam?.numberOfCorrectionRoundsInExam == 2 && exercise.value?.baseExercise.secondCorrectionEnabled == true
+    }
+    
+    var currentCorrectionRound: CorrectionRound {
+        (isSecondCorrectionRoundEnabled && assessed == 1.0) ? .second : .first
```

Comment:
The `assessed == 1.0` looks a bit sketchy. Isn't there a better way to check this? 🤔

Reply:
I've looked into it but couldn't come up with a better solution. The web client sends a request for each round to get the next submission without locking it every time the dashboard is opened to determine whether the first correction round is over. I can implement the same, but I'm not sure if sending 2 additional network requests is a good idea 🤔

Reply:
Ok, then we leave it for now